gradle/libs.versions.toml

```diff
@@ -14,9 +14,8 @@ moshi = "1.15.1"
 minSdk = "25"
 compileSdk = "34"
 
-# Maps to this commit: https://android.googlesource.com/platform/prebuilts/studio/layoutlib/+/6c7316c
-layoutlib = "2023.2.1-6c7316c"
-layoutlibPrebuiltSha = "6c7316c"
+layoutlibApi = "31.3.1"
```

Comment:
Should this be replaced with versions.androidTools instead?

Reply:
I've kept it separate like before because there might still be situations in the future where this might need to be updated independently.